src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -315,8 +315,6 @@ void sizeZoneSpaceEquipmentPart1(EnergyPlusData &state,
                                                 : state.dataZoneTempPredictorCorrector->zoneHeatBalance(zoneNum).NonAirSystemResponse;
     auto &sysDepZoneLoads = (spaceNum > 0) ? state.dataZoneTempPredictorCorrector->spaceHeatBalance(spaceNum).SysDepZoneLoads
                                            : state.dataZoneTempPredictorCorrector->zoneHeatBalance(zoneNum).SysDepZoneLoads;
-    auto &zoneLatentGain = (spaceNum > 0) ? state.dataZoneTempPredictorCorrector->spaceHeatBalance(spaceNum).latentGain
-                                          : state.dataZoneTempPredictorCorrector->zoneHeatBalance(zoneNum).latentGain;
```

Comment:
[src/EnergyPlus/ZoneEquipmentManager.cc:318]:(style),[variableScope],The scope of the variable 'zoneLatentGain' can be reduced.

Reply:
@mjwitte this is 1 thing causing a conflict in this branch.  It looks like you did something different with zoneLatentGain maybe?  I'm hesitant to smash in a conflict resolution when there may be side effects.

Reply:
This can go all the way at the bottom of the function inside this:
```
        if (zsCalcSizing.zoneLatentSizing) {
            int ZoneMult = zoneOrSpace.Multiplier * zoneOrSpace.ListMultiplier;
           auto &zoneLatentGain = (spaceNum > 0) ? state.dataZoneTempPredictorCorrector>spaceHeatBalance(spaceNum).latentGain
                                          : state.dataZoneTempPredictorCorrector->zoneHeatBalance(zoneNum).latentGain;
            zoneLatentGain += (LatOutputProvided * HgAir) / ZoneMult;
        }
```